<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reports</class>
 <widget class="QDialog" name="Reports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 245, 245)</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Число:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dayComboBox">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Месяц:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="monthComboBox">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Год:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="yearComboBox">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="temporaryLabel">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Работник:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="temporaryComboBox">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateReportButton">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(167, 255, 162);
outline: none;</string>
       </property>
       <property name="text">
        <string>Сформировать отчёт</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="workButton">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(167, 255, 162);
outline: none;</string>
       </property>
       <property name="text">
        <string>Отчёт о работе сотрудника</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="servicesButton">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(167, 255, 162);
outline: none;</string>
       </property>
       <property name="text">
        <string>Отчёт об услуге</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cashButton">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(167, 255, 162);
outline: none;</string>
       </property>
       <property name="text">
        <string>Доход салона</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ExitButton">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(167, 255, 162);
outline: none;</string>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="styleSheet">
        <string notr="true">background-color: white;
</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="resultLineEdit">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
